src/data/prayers/data-source.js

```diff
@@ -312,7 +312,7 @@ export default class Prayer extends RockApolloDataSource {
       return this.getFromId(id);
     } catch (e) {
       bugsnagClient.notify(new Error('Answering prayer failed.'), {
-        metaData: { rockPrayerID: id, answer },
```

Comment:
Is there a reason you took this out? seems like it may be useful

Reply:
Nope. Just totally missed putting `answer` back when I put this function back the way it was. Fixed now.